CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ documentation, we greatly value feedback and contributions from our community.
 Following these guidelines helps smooth out the process of contributing for both you as a contributor and those who maintain the project, making better use of everyone's time and energy.
 
 ### Reporting a bug
-If you find a bug in Xfer, please file an issue using our [template](issue template), detailing as much about your setup as possible to help future developers in reproducing the issue. Tag it as a bug, with any additional tags as relevant.
+If you find a bug in Xfer, please file an issue, detailing as much about your setup as possible to help future developers in reproducing the issue. Tag it as a bug, with any additional tags as relevant.
```

Comment:
Instead of removing the reference to the issue template, should we actually make an issue template?